.github/workflows/ci.yml

```diff
@@ -274,6 +274,16 @@ jobs:
               docker compose exec -it client timeout 60 \
               sh -c 'until ping -W 1 -c 1 172.20.0.100 &>/dev/null; do true; done'
 
+          # Systemd-related tests (ignores the client container and runs in the runner itself)
+          - test_name: systemd-resolved
+            setup: scripts/systemd-test-setup.bash
+            execute: |
+              resolvectl status
+              sudo systemctl start firezone-client
+              # TODO: Something better than a sleep
```

Comment:
Is it worth using [`sd_notify`](https://www.freedesktop.org/software/systemd/man/latest/sd_notify.html#)?